web/custom-plugins/forecast-of-response/src/Types.ts

```diff
@@ -0,0 +1,41 @@
+export type TRoute = {
+  path: string
+  content: string
+}
+
+export type TSettings = {
+  name: string
+  label: string
+  tabIndex: number
+  hidden: boolean
+  visibleDataSources: any
+  type: string
+  description: string
+  packages: any
+  models: any
+  actions: any
+  file_loc: string
+  data_source_aliases: any
+}
+
+export type TApp = {
+  applications: any
+  settings: TSettings
+}
+
+export type TAppHeader = {
+  appName: string
+}
```

Comment:
Interfaces which takes only one simple type as argument can have it's type decleration directly in the function declaration